### PR TITLE
[3.4.2] Replace @Scheduled annotations with Scheduled Service

### PR DESCRIPTION
**Pull request by irynamatveieva**
## Pull Request description

Issue:#6529 

## General checklist

- [x] You have reviewed the guidelines [document](https://docs.google.com/document/d/1wqcOafLx5hth8SAg4dqV_LV3un3m5WYR8RdTJ4MbbUM/edit?usp=sharing).
- [x] PR name contains fix version. For example, "[3.3.4] Hotfix of some UI component" or "[3.4] New Super Feature".
- [x] Description references specific [issue](https://github.com/thingsboard/thingsboard/issues).
- [x] Description contains human-readable scope of changes.
- [x] Description contains brief notes about what needs to be added to the documentation.
- [x] No merge conflicts, commented blocks of code, code formatting issues.
- [x] Changes are backward compatible or upgrade script is provided.
- [ ] Similar PR is opened for PE version to simplify merge. Required for internal contributors only.
  
## Front-End feature checklist

- [x] Screenshots with affected component(s) are added. The best option is to provide 2 screens: before and after changes;
- [x] If you change the widget or other API, ensure it is backward-compatible or upgrade script is present.
- [x] Ensure new API is documented [here](https://github.com/thingsboard/thingsboard-ui-help)

## Back-End feature checklist

- [x] Added corresponding unit and/or integration test(s). Provide written explanation in the PR description if you have failed to add tests.
- [x] If new dependency was added: the dependency tree is checked for conflicts.
- [x] If new service was added: the service is marked with corresponding @TbCoreComponent, @TbRuleEngineComponent, @TbTransportComponent, etc.
- [x] If new REST API was added: the RestClient.java was updated, issue for [Python REST client](https://github.com/thingsboard/thingsboard-python-rest-client) is created.

### CODE REVIEW

**Review by ShvaykaD — APPROVED** [review]

File: application/src/main/java/org/thingsboard/server/service/queue/DefaultTbCoreConsumerService.java [hunk 1]
```diff
@@ -117,6 +119,7 @@ public class DefaultTbCoreConsumerService extends AbstractConsumerService<ToCore
     private final TbCoreConsumerStats stats;
     protected final TbQueueConsumer<TbProtoQueueMsg<ToUsageStatsServiceMsg>> usageStatsConsumer;
     private final TbQueueConsumer<TbProtoQueueMsg<ToOtaPackageStateServiceMsg>> firmwareStatesConsumer;
+    private final TbPrintStatsExecutorService tbPrintStatsExecutorService;
```
Comment (ashvayka) [inline application/src/main/java/org/thingsboard/server/service/queue/DefaultTbCoreConsumerService.java:122]: Move this variable to the super class (AbstractConsumerService)

File: application/src/main/java/org/thingsboard/server/service/queue/DefaultTbRuleEngineConsumerService.java [hunk 2]
```diff
@@ -103,6 +105,7 @@ public class DefaultTbRuleEngineConsumerService extends AbstractConsumerService<
     private final ConcurrentMap<String, TbRuleEngineQueueConfiguration> consumerConfigurations = new ConcurrentHashMap<>();
     private final ConcurrentMap<String, TbRuleEngineConsumerStats> consumerStats = new ConcurrentHashMap<>();
     private final ConcurrentMap<String, TbTopicWithConsumerPerPartition> topicsConsumerPerPartition = new ConcurrentHashMap<>();
+    private final TbPrintStatsExecutorService tbPrintStatsExecutorService;
```
Comment (ashvayka) [inline application/src/main/java/org/thingsboard/server/service/queue/DefaultTbRuleEngineConsumerService.java:108]: Move this variable to the super class (AbstractConsumerService)

File: common/stats/src/main/java/org/thingsboard/server/common/stats/DefaultTbPrintStatsExecutorService.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import org.thingsboard.common.util.ThingsBoardThreadFactory;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+@Component
+public class DefaultTbPrintStatsExecutorService implements TbPrintStatsExecutorService {
```
Comment (ashvayka) [inline common/stats/src/main/java/org/thingsboard/server/common/stats/DefaultTbPrintStatsExecutorService.java:28]: Please consider merging this component with the ScheduledLogExecutorComponent. And also use the same component in the CassandraBufferedRateReadExecutor/WriteExecutor

File: dao/src/main/java/org/thingsboard/server/dao/sqlts/timescale/TimescaleTimeseriesDao.java [hunk 4]
```diff
@@ -44,6 +44,7 @@
 
 import javax.annotation.PostConstruct;
 import javax.annotation.PreDestroy;
+import java.util.*;
```
Comment (ShvaykaD) [inline dao/src/main/java/org/thingsboard/server/dao/sqlts/timescale/TimescaleTimeseriesDao.java:47]: @irynamatveieva please remove import wildcards

File: dao/src/main/java/org/thingsboard/server/dao/JpaDaoConfig.java [hunk 5]
```diff
@@ -27,6 +27,7 @@
  */
 @Configuration
 @TbAutoConfiguration
+@ComponentScan({"org.thingsboard.server.dao.sql", "org.thingsboard.server.common.stats"})
```
Comment (ShvaykaD) [inline dao/src/main/java/org/thingsboard/server/dao/JpaDaoConfig.java:30]: @irynamatveieva please fix the duplicate of component scan

**Comment by ashvayka** [comment]
Please fix the conflicting files

File: dao/src/main/java/org/thingsboard/server/dao/sqlts/timescale/TimescaleTimeseriesDao.java [hunk 6]
```diff
@@ -174,7 +174,8 @@ private ListenableFuture<List<TsKvEntry>> findAllAsyncWithLimit(EntityId entityI
                 query.getStartTs(),
                 query.getEndTs(),
                 PageRequest.of(0, query.getLimit(),
-                        Sort.by(new Sort.Order(Sort.Direction.fromString(query.getOrder()),  "ts").nullsNative())));;
+                        Sort.by(new Sort.Order(Sort.Direction.fromString(query.getOrder()), "ts").nullsNative())));
```
Comment (ShvaykaD) [inline dao/src/main/java/org/thingsboard/server/dao/sqlts/timescale/TimescaleTimeseriesDao.java:177]: @irynamatveieva please remove the redundant semicolon

**Review by ShvaykaD — APPROVED** [review]

**Review by ShvaykaD — APPROVED** [review]

**Pull request closed without merging**